virkarekisteri/virkarekisteri
# VIR-118 Sijaisuus

Added feature to add Sijainen (Replacement)

How to test:
Add a PositionEmployee
Modify the employee, see the Toggle to activate replacement
Check that it is updated correctly
